### PR TITLE
[18.09] backport xss

### PR DESCRIPTION
**Pull request by martenson**
Backport of sanity-PR #7616 .

### CODE REVIEW

**Review by nsoranzo — APPROVED** [review]

**Comment by martenson** [comment]
the selenium fails are relevant here, I will push a fix

File: templates/message.mako [hunk 1]
```diff
@@ -51,12 +53,11 @@
 
 ## Render large message.
 <%def name="render_large_message( message, status )">
-    <div class="${status}messagelarge" style="margin: 1em">${_(message)}</div>
+    <div class="${bleach.clean(status)}messagelarge" style="margin: 1em">${_(bleach.clean(message))}</div>
```
Comment (nsoranzo) [inline templates/message.mako]: [Bleach documentation](https://bleach.readthedocs.io/en/latest/clean.html) says it is not safe to use `clean()` for HTML attributes, which is why I implemented the previous validation approach.

**Review by jmchilton — APPROVED** [review]

File: templates/message.mako (hunk 1 above)
Comment (martenson) [reply templates/message.mako]: thanks, I've pushed a rebase

**Review by nsoranzo — APPROVED** [review]

**Comment by nsoranzo** [comment]
Thanks @martenson !

**Pull request merged**